pkgs/build-support/trivial-builders/default.nix

```
@@ -9,31 +9,35 @@ in
 
 rec {
 
-  /* Run the shell command `buildCommand' to produce a store path named
-   `name'.  The attributes in `env' are added to the environment
-   prior to running the command. By default `runCommand` runs in a
-   stdenv with no compiler environment. `runCommandCC` uses the default
-   stdenv, `pkgs.stdenv`.
+  /* Run the shell command `buildCommand' to produce a store path named `name'.
```

5 spaces of indentation is editor-unfriendly.

```suggestion
  /*
    Run the shell command `buildCommand' to produce a store path named `name'.
```